posts/2021-02-19-mpcontextpropagation-requesttiming-21003-beta.adoc

```diff
@@ -0,0 +1,314 @@
+---
+layout: post
+title: "RequestTiming 1.0, MicroProfile Context Propagation 1.2, enhancments to App Manager configuration and Liberty connection management in Open Liberty 21.0.0.3-beta"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: RequestTiming 1.0, MicroProfile Context Propagation 1.2, enhancments to App Manager configuration and Liberty connection management in Open Liberty 21.0.0.3-beta - OpenLiberty.io
+seo-description: Open Liberty 21.0.0.3-beta contains four All Beta features (new `expandLocation` property for App Manager configuration, Ability to control whether thread dumps are collected when a hung request is detected in the Request Timing feature (RequestTiming 1.0), MicroProfile Context Propagation 1.2 and Automatic Cleanup of Leaked Connections) and one feature in Jakarta EE 9 (JakartaMail).
+blog_description: Open Liberty 21.0.0.3-beta contains four All Beta features (new `expandLocation` property for App Manager configuration, Ability to control whether thread dumps are collected when a hung request is detected in the Request Timing feature (RequestTiming 1.0), MicroProfile Context Propagation 1.2 and Automatic Cleanup of Leaked Connections) and one feature in Jakarta EE 9 (JakartaMail). Also in 21.0.0.3-beta the link:https://github.com/OpenLiberty/open-liberty/issues/15649[issue in MicroProfile Fault Tolerance 3.0] from 21.0.0.2-beta has now been resolved meaning that the `mpFaultTolerance-3.0` feature and the `microProfile-4.0` convenience feature can be used in the Open Liberty 21.0.0.3-beta.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= RequestTiming 1.0, MicroProfile Context Propagation 1.2, enhancments to App Manager configuration and Liberty connection management in Open Liberty 21.0.0.3-beta
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+
+Open Liberty 21.0.0.3-beta contains four All Beta features (new `expandLocation` property for App Manager configuration, Ability to control whether thread dumps are collected when a hung request is detected in the Request Timing feature (RequestTiming 1.0), MicroProfile Context Propagation 1.2 and Automatic Cleanup of Leaked Connections) and one feature in Jakarta EE 9 (JakartaMail). Also in 21.0.0.3-beta, the link:https://github.com/OpenLiberty/open-liberty/issues/15649[issue in MicroProfile Fault Tolerance 3.0] from 21.0.0.2-beta has now been resolved meaning that the `mpFaultTolerance-3.0` feature and the `microProfile-4.0` convenience feature can be used in the Open Liberty 21.0.0.3-beta.
```

Comment:
```suggestion
Open Liberty 21.0.0.3-beta contains four new "All Beta" features and one new Jakarta EE 9 feature. We have also resolved the link:https://github.com/OpenLiberty/open-liberty/issues/15649[MicroProfile Fault Tolerance 3.0 issue] from 21.0.0.2-beta which means that the `mpFaultTolerance-3.0` feature and the `microProfile-4.0` convenience feature can be properly used in the Open Liberty 21.0.0.3-beta.
```

Reply:
Changed line 19 to "Open Liberty 21.0.0.3-beta contains four new "All Beta" features and one new Jakarta EE 9 feature. Also in 21.0.0.3-beta, all the MicroProfile 4.0 features are now available."

Reply:
@tevans78

Reply:
```suggestion
In addtion to all the Jakarta EE 9 [link] and MicroProfile 4.0 [link] features which were in 21.0.0.2-beta, Open Liberty 21.0.0.3-beta also contains four new "All Beta" features and one new Jakarta EE 9 feature.
```

Reply:
Done